packages/react-core/src/components/DragDrop/examples/DragDropMultipleLists.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { DragDrop, Draggable, Droppable, Split, SplitItem } from '@patternfly/react-core';
+
+interface ItemType {
+  id: string;
+  content: string;
+}
+
+interface SourceType {
+  droppableId: string;
+  index: number;
+}
+
+interface DestinationType extends SourceType {}
+
+const getItems = (count: number, startIndex?: number) =>
```

Comment:
Looks like this line is causing the syntax error in the preview build, the `?` in particular I believe?

![Multiple lists example syntax error](https://user-images.githubusercontent.com/70952936/163831595-c02863e5-7a8f-48fa-ade9-67e1a286273e.png)